docarray/doc_index/abstract_doc_index.py

```diff
@@ -39,7 +39,7 @@
 
 
 class FindResultBatched(NamedTuple):
-    documents: List[DocumentArray]
+    documents: Union[List[DocumentArray], List[List[Dict[str, Any]]]]
```

Comment:
the `FindResultBatched` is the thing that the user gets in the end, and us such it should always carry Documents, not dicts. So i wold undo that change